content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -131,6 +133,20 @@ Once you see the Metric Stream resource has been successfully created, wait five
 {{% /tab %}}
 {{< /tabs >}}
 
+### Disabling Metric Streaming
+
+In order to disable metric streaming completely for a given AWS account and region, you must delete the AWS Metric Steam and its related resources. In order to prevent loss of metrics in Datadog, it's important to follow these deletion steps carefully:
+
+If you set streaming up with <a href="https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/?tab=cloudformation#installation">CloudFormation</a>:
+  1. Delete the stack that was created during the setup
+
+If you set streaming up through the <a href="https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/?tab=awsconsole#installation">AWS Console</a>:
+  1. Delete the Kinesis Data Firehose delivery stream that delivered metrics to Datadog
+  2. Delete the CloudWatch Metric Stream linked to your delivery stream
+  3. Delete the IAM roles associated with the stream and all other resources that were created while setting up the stream 
```

Comment:
@rmshree 

- Make deleting the CloudWatch Metric Stream the first step (that is what we look at to start polling again)
- add another step after the Kinesis Data Firehose for deleting the backup s3 bucket for failed messages linked to the Firehose.